pkg/application/create.go

```diff
@@ -122,16 +271,24 @@ func ValidateCreate(op CreateOp, githubClient *github.Client) error {
 	if err != nil {
 		return fmt.Errorf("%s: %v", op.LocalPath, err)
 	}
-	_, response, err := githubClient.Repositories.Get(
-		context.Background(),
-		op.OrgName,
-		op.Name,
-	)
-	if err == nil {
-		return fmt.Errorf("%s/%s repository is already exists", op.OrgName, op.Name)
+
+	_, isMonorepo, err := openMonorepoMaybe(op.LocalPath)
+	if err != nil {
+		return fmt.Errorf("checking for monorepo failed with %v", err)
 	}
-	if err != nil && response.StatusCode != http.StatusNotFound {
-		return fmt.Errorf("error while checking if %s/%s repository exists", op.OrgName, op.Name)
+
+	if isMonorepo {
+		_, response, err := githubClient.Repositories.Get(
+			context.Background(),
+			op.OrgName,
+			op.Name,
+		)
+		if err == nil {
+			return fmt.Errorf("%s/%s repository already exists", op.OrgName, op.Name)
+		}
+		if err != nil && response.StatusCode != http.StatusNotFound {
```

Comment:
probably don't need err != nil here, it'll always be not nil if it gets here

Reply:
haha, not sure how this get here